CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ can be found at [#317](https://github.com/grafana/agent/issues/317).
 
 # Master (unreleased)
 
+- [ENHANCEMENT] Only keep a handful of K8s API server metrics by default to reduce
+  default active series usage.
+
```

Comment:
I try to sort the changelog, can you add it after the last `[ENHANCEMENT]` entry and also credit yourself like the other examples?

Reply:
yep, sorry